warehouse/forklift/legacy.py

```diff
@@ -124,17 +124,19 @@ def namespace_stdlib_list(module_list):
     "linux_armv7l",
 }
 # macosx is a little more complicated:
-_macosx_platform_re = re.compile(r"macosx_10_(\d+)_(?P<arch>.*)")
+_macosx_platform_re = re.compile(r"macosx_(?P<major>\d+)_(\d+)_(?P<arch>.*)")
 _macosx_arches = {
     "ppc",
     "ppc64",
     "i386",
     "x86_64",
+    "arm64",
```

Comment:
arm64, universal2 discussed [here](https://discuss.python.org/t/apple-silicon-and-packaging/4516/21), implemented [here](https://github.com/pypa/packaging/pull/319).